packages/babel-helpers/src/helpers.js

```diff
@@ -859,10 +859,16 @@ helpers.arrayWithHoles = helper("7.0.0-beta.0")`
 `;
 
 helpers.iterableToArray = helper("7.0.0-beta.0")`
+
   export default function _iterableToArray(iter) {
     if (
-      Symbol.iterator in Object(iter) ||
-      Object.prototype.toString.call(iter) === "[object Arguments]"
+      Array.isArray(iter)
+      || typeof iter === 'string'
+      || (typeof global.Symbol === 'function' && Symbol.iterator in Object(iter))
```

Comment:
`global` is a standard thing in node; but sure, using real globals is fine if the transform can skip them.

In general, i think any global in a `typeof` check shouldn't be transformed, and anything guarded by that check also shouldn't be.

Reply:
Instead of using `global.Symbol` or `toString`, I made an update to skip pulling in shim if we are checking for `typeof Symbol === 'function'`: e0111425f9603fe5259812611db3dae9469e5561 

Would something like this work? it would be a way to unblock those running into this problem on on ie11 without bringing in the shim for Symbol. 

I removed the checks for `Map` and `Set` for now, we can move that to a separate PR but this will help unblock those who are using babel with ie11 without a shim for Symbol.